client/data/settings/hooks.js

```diff
@@ -27,18 +27,30 @@ export const useEnabledPaymentMethodIds = () => {
 };
 
 export const useGeneralSettings = () => {
+	const { updateAccountStatementDescriptor } = useDispatch( STORE_NAME );
```

Comment:
(minor) it looks like in the other cases there is hook with a `[value, updateValue]` pair for each setting.
Rather than having a `useGeneralSettings` hooks, what are people's thoughts on keeping on with the "convention" of having a separate hook in the format of `const [value, setValue] = useGetValue()` for each setting?

Not saying that it needs to change in this PR - after all this hook is called `useGeneralSettings`, I think it makes sense to keep the `statementDescriptor` and the `manualCapture` fields here - just considering it for the future.

My reasoning for this is that if the `accountStatementDescriptor` (or any other field) is moved from the "General settings" UI, we would need to also refactor this.

Reply:
Agree with you, sir. I prefer the consistency of atomic setters/getters.

Reply:
Done: https://github.com/Automattic/woocommerce-payments/pull/1889